hacks_mbof/urls.py

```diff
@@ -43,7 +45,20 @@
     url(r'^bower_components/(?P<path>.*)$', serve, {
         'document_root': 'mbofui/bower_components',
     }),
+]
+
+if 'djangosaml2' in settings.INSTALLED_APPS:
+    urlpatterns += (
+        url(r'^accounts/', include('djangosaml2.urls')),
+        url(r'^test/', 'djangosaml2.views.echo_attributes'),
+    )
```

Comment:
Ah, I see what you are saying. I was misunderstanding. To be clear, in the first declaration we have

urlpatterns = [ ... ]

and in the second part we have

urlpatterns += ( (x1,y1),(x2,y2) )

Rather than urlpatterns += [ ... ] or what I've seen in the past: urlpatterns.append( ... ) or urlpatterns.extend( ... )

Interesting point. I'll investigate this.